pages/home.html

```diff
@@ -22,7 +26,7 @@ <h1 class="list-title"><a href="/features/">Latest features</a></h1>
 			</li>
 			{% endfor %}
 		</ol>
-		<!-- <a href="/features/" class="button">View all features</a> -->
+		<a href="/features/" class="button">View all features</a>
```

Comment:
How about copying the about button in the navbar like on caniuse?
https://caniuse.com/ciu/about

The about page should probably then describe the site. We can link to the usages and challenges doc for the moment for folk who want to learn more so that we don't have to maintain the definition in both places:
https://webview-cg.github.io/usage-and-challenges/#types-of-webviews

Reply:
I like the idea of having a simpler definition on the site easily accessible. Redirecting to the "bulky" description in the draft is not user friendly IMHO. Of course this is subject to change once we have more documentation.

I made some adjustments and added the about button. WDYT?

Reply:
I have no worries iterating as we go along! Happy to move ahead as is :)